zircon.jvm.examples/src/main/kotlin/org/hexworks/zircon/examples/playground/TableOld.kt

```diff
@@ -4,7 +4,7 @@ import org.hexworks.zircon.api.Components
 import org.hexworks.zircon.api.component.Fragment
 import org.hexworks.zircon.api.data.Size
 
-class Table<M : Any>(
+class TableOld<M : Any>(
```

Comment:
Poor old table. Will it ever be young again?

Reply:
I will carry it to its grave ⚰
Just forgot to do it yet.

Reply:
Done with 9870759f